api/src/main/java/com/codeforcommunity/exceptions/WrongFavoriteStatusException.java

```diff
@@ -4,22 +4,22 @@
 import io.vertx.ext.web.RoutingContext;
 
 public class WrongFavoriteStatusException extends HandledException {
-    private Boolean isAlreadyFavorite;
+  private Boolean isAlreadyFavorite;
```

Comment:
Is this from running spotless?

Reply:
Yes

Reply:
I know it’s usually bad practice to include unrelated changes in a PR; I believe spotless was run automatically when I ran mvn clean install. I can separate them into another PR if you want.

Reply:
That's my bad, keep this in here. Not sure why my spotless didn't pick this up, maybe I forgot to run it